Kinvey/Kinvey/Geolocation.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  Geolocation.swift
+//  Kinvey
+//
+//  Created by Victor Hugo on 2017-02-01.
+//  Copyright © 2017 Kinvey. All rights reserved.
+//
+
+import Foundation
+import RealmSwift
+import ObjectMapper
+import CoreLocation
+import MapKit
+
+open class GeoPoint: Object, StaticMappable {
+    
+    dynamic var latitude: CLLocationDegrees = 0.0
+    dynamic var longitude: CLLocationDegrees = 0.0
+    
+    public static func objectForMapping(map: Map) -> BaseMappable? {
+        guard let _: Double = map["latitude"].value(), let _: Double = map["longitude"].value() else {
+            return nil
+        }
+        return GeoPoint()
+    }
+    
+    public convenience init(latitude: Double, longitude: Double) {
+        self.init()
+        self.latitude = latitude
+        self.longitude = longitude
+    }
+    
+    public convenience init(coordinate: CLLocationCoordinate2D) {
+        self.init()
+        self.latitude = coordinate.latitude
+        self.longitude = coordinate.longitude
+    }
+    
+    public convenience init(location: CLLocation) {
+        self.init()
+        self.latitude = location.coordinate.latitude
+        self.longitude = location.coordinate.longitude
+    }
+    
+    public convenience init(_ array: [Double]) {
```

Comment:
Same comment as above. Exposing this constructor adds unnecessary work e.g. we have to document which value in the array is lat and which is long, etc.